bundles/remote_services/topology_manager/src/topology_manager.c

```diff
@@ -209,7 +209,6 @@ celix_status_t topologyManager_rsaAdded(void * handle, service_reference_pt unus
 
 	celixThreadMutex_lock(&manager->rsaListLock);
 	arrayList_add(manager->rsaList, rsa);
-    celixThreadMutex_unlock(&manager->rsaListLock);
 
 	// add already imported services to new rsa
     celixThreadMutex_lock(&manager->importedServicesLock);
```

Comment:
Then we have to make sure ABBA deak-lock does not happen: `rsaListLock` should always be acquired before `importedServicesLock`.